tests/test_serializable_request.py

```diff
@@ -67,8 +67,8 @@ async def client():
     #   TODO command arg for subnet_tag
     session = Session(EXECUTOR_CFG)
     session.startup(**STARTUP_CFG)(echo_server_startup)
-    async with session.client() as client:
-        yield client
+    async with session.client() as cli:
```

Comment:
`cli` is a popular acronym for 'command line interface' so it's not the best choice in my opinion. What was wrong with `client`?

Reply:
It's the same as the function name and `pylint` doesn't like this:
```
tests/test_serializable_request.py:70:35: W0621: Redefining name 'client' from outer scope (line 66) (redefined-outer-name)
```

Reply:
Oh, okay then.